Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -932,25 +939,33 @@ def genS(fragment,howMuch):
 hiAlca2017 = {'--conditions':'auto:phase1_2017_realistic', '--era':'Run2_2017_pp_on_XeXe'}
 hiAlca2018 = {'--conditions':'auto:phase1_2018_realistic', '--era':'Run2_2018'}
 hiAlca2018_ppReco = {'--conditions':'auto:phase1_2018_realistic_hi', '--era':'Run2_2018_pp_on_AA'}
+hiAlca2021_ppReco = {'--conditions':'auto:phase1_2021_realistic_hi', '--era':'Run3_PbPb'}
 
 
 hiDefaults2011=merge([hiAlca2011,{'--scenario':'HeavyIons','-n':2}])
 hiDefaults2015=merge([hiAlca2015,{'--scenario':'HeavyIons','-n':2}])
 hiDefaults2017=merge([hiAlca2017,{'-n':2}])
 hiDefaults2018=merge([hiAlca2018,{'--scenario':'HeavyIons','-n':2}])
 hiDefaults2018_ppReco=merge([hiAlca2018_ppReco,{'-n':2}])
+hiDefaults2021_ppReco=merge([hiAlca2021_ppReco,{'-n':2}])
 
 steps['HydjetQ_B12_5020GeV_2011']=merge([{'-n':1,'--beamspot':'RealisticHI2011Collision'},hiDefaults2011,genS('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 steps['HydjetQ_B12_5020GeV_2015']=merge([{'-n':1,'--beamspot':'RealisticHICollisionFixZ2015'},hiDefaults2015,genS('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 steps['HydjetQ_MinBias_XeXe_5442GeV_2017']=merge([{'-n':1},hiDefaults2017,gen2017('Hydjet_Quenched_MinBias_XeXe_5442GeV_cfi',U2000by1)])
 steps['HydjetQ_B12_5020GeV_2018']=merge([{'-n':1},hiDefaults2018,gen2018prod('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 steps['HydjetQ_B12_5020GeV_2018_ppReco']=merge([{'-n':1},hiDefaults2018_ppReco,gen2018hiprod('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
+steps['HydjetQ_B12_5020GeV_2021_ppReco']=merge([{'-n':1},hiDefaults2021_ppReco,gen2021hiprod('Hydjet_Quenched_B12_5020GeV_cfi',U2000by1)])
 
 steps['QCD_Pt_80_120_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('QCD_Pt_80_120_13TeV_TuneCUETP8M1_cfi',Kby(9,150))])
 steps['PhotonJets_Pt_10_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('PhotonJet_Pt_10_13TeV_TuneCUETP8M1_cfi',Kby(9,150))])
 steps['ZMM_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('ZMM_13TeV_TuneCUETP8M1_cfi',Kby(18,100))])
 steps['ZEEMM_13_HI']=merge([hiDefaults2018_ppReco,gen2018hiprod('ZEEMM_13TeV_TuneCUETP8M1_cfi',Kby(18,300))])
 
+steps['QCD_Pt_80_120_13_HI_2021']=merge([hiDefaults2021_ppReco,gen2021hiprod('QCD_Pt_80_120_13TeV_TuneCUETP8M1_cfi',Kby(9,150))])
+steps['PhotonJets_Pt_10_13_HI_2021']=merge([hiDefaults2021_ppReco,gen2021hiprod('PhotonJet_Pt_10_13TeV_TuneCUETP8M1_cfi',Kby(9,150))])
+steps['ZMM_13_HI_2021']=merge([hiDefaults2021_ppReco,gen2021hiprod('ZMM_13TeV_TuneCUETP8M1_cfi',Kby(18,100))])
```

Comment:
@chayanit  Yes, it would make sense to switch to 14 TeV.  Now that it occurs to me, it would be very useful to have one gen fragment that's run with both the standard pp settings and the HI era.   Would it be ok to add a TTbar_14TeV_TuneCP5 workflow with heavy-ion settings  for this purpose?   Regarding the other point, I will clean up the ZMM, which is no longer used.

Reply:
@mandrenguyen we do have these GEN fragments for 14TeV. Please update
https://github.com/cms-sw/cmssw/blob/master/Configuration/Generator/python/QCD_Pt_80_120_14TeV_TuneCUETP8M1_cfi.py
https://github.com/cms-sw/cmssw/blob/master/Configuration/Generator/python/PhotonJet_Pt_10_14TeV_TuneCUETP8M1_cfi.py
but not ZEEMM. Would you like to switch from ZEEMM to TTbar?

Reply:
Thanks!  Z bosons tend to be pretty useful for us, I'm not sure I would like get rid of these wfs.   I see that there are separate ZMM and ZEE gen fragments for 14 TeV already.  What if  switched to these and reduced the number of events for each wf by a factor of 2?  
While the 14 TeV fragments exist, are any of these workflows actually run in the IB?    That was my idea for adding ttbar.  On the other hand, the scope of this PR is already spreading beyond my initial intention, so maybe we can drop that idea.

Reply:
@mandrenguyen  Instead of split them to two more workflows, I would recommend to create new GEN fragment for ZEEMM 14TeV under Configuration/Generator/python. You can just copy this GEN fragment and update energy to 14TeV https://github.com/cms-sw/cmssw/blob/master/Configuration/Generator/python/ZEEMM_13TeV_TuneCUETP8M1_cfi.py
I don't think any of these workflows are included in the IB, usually we use TTbar. You can try to add and I will have a look. 

I have other comments elsewhere. Please also consider the changes. Thanks.

Reply:
@chayanit Your other comments seem straightforward, so I will address them as soon as I can find some time :-)